content/gamenews

```diff
@@ -1,8 +1,8 @@
+2017-08-27 20-00-00:
+	Title: Playtest 20170827 available!
+	Author: Reaperrr
+	Content: We're pleased to announce the new playtest-20170827, the second release candidate for the upcoming OpenRA release. Changes include:\n\n  • Over 30 bugs and other issues fixed\n  • New civilian tech structures and a new map for the RA mod\n  • Improved Red Alert balance\n  • Fine-tuned Tiberian Dawn balance\n  • Updated Mod SDK\n\nHead to www.openra.net for more information!
```

Comment:
The second bullet point overflows onto a second line.  Can you change it to just "...and a new map for RA"?